libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1072,6 +1076,13 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         tripSession.unregisterStateObserver(tripSessionStateObserver)
     }
 
+    /**
+     * Set the [TripNotificationInterceptor]
```

Comment:
Could you clarify that this applied only to our internal, default implementation and that `null` clears the interceptor?